corehq/const.py

```diff
@@ -28,3 +28,10 @@
 
 ONE_DAY = 60 * 60 * 24
 DEFAULT_PARALLEL_EXECUTION_TIMEOUT = 60 * 60  # one hour
+
+
+API_REGISTRATION = "api"
+WEB_REGISTRATION = "web"
+SMS_REGISTRATION = "sms"
+BULK_IMPORTER = "bulk_importer"
+INVITATION_REGISTRATION = "invitation"
```

Comment:
Shouldn't these be in the `users` app? Also perhaps rename these with a consistent prefix: `USER_CHANGE_SOURCE_` or something. Or else make a class:
```python
class UserChangeVia:
    api_registration = 'api'
    ...
```

Reply:
I thought this const didn't belong to any app because no single app cover all these, so i put them in the out most scope of corehq.
Sure, will rename.

Reply:
https://github.com/dimagi/commcare-hq/pull/27755/commits/bd47bd9fabb0b270f255225a4eab64f310980038